src/core/paginatedEmbed.js

```diff
@@ -0,0 +1,98 @@
+const { MessageActionRow } = require('discord.js');
+
+/**
+ * Makes it easy to build paginated embeds.
+ * 
+ * Forked from https://www.npmjs.com/package/discordjs-button-pagination and
+ * modified to support custom footers and non-reply messages.
+ */
+const paginatedEmbed = async (
+    message,
+    pages,
+    buttons, {
+    timeout = 120000,
+    useDefaultFooter = true,
+    useReplies = false,
+} = {}) => {
+    if (!message && !message.channel) throw new Error('Channel is inaccessible.');
+    if (!pages) throw new Error('Pages are not given.');
+    if (!buttons) throw new Error('Buttons are not given.');
+    if (buttons.length !== 2) throw new Error('Need two buttons.');
+    if (buttons[0].style === 'LINK' || buttons[1].style === 'LINK') {
+        throw new Error('Link buttons are not supported.');
+    }
```

Comment:
No single line condition.

To avoid multiline function declaration, you can do destructuring and defaulting inside the function.

```suggestion
const paginatedEmbed = async (message, pages, buttons, options = {}) => {

	const { timeout = 120000, useDefaultFooter = true, useReplies = false } = options;

    if (!message && !message.channel) {

    	throw new Error('Channel is inaccessible.');
    }

    if (!Array.isArray(pages)) {

    	throw new Error('Pages are not given.');
    }

    if (!Array.isArray(buttons)) {

    	throw new Error('Buttons are not given.');
    }

    if (buttons.length !== 2) {

    	throw new Error('Need two buttons.');
    }

    if (buttons.some(({ style }) => style === 'LINK')) {

        throw new Error('Link buttons are not supported.');
    }
```

Reply:
Ugh. I don't like single like conditions either but the alternative looks disgusting here. I guess I'll do it tho, altho I won't have empty lines in between. 🙂